wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py

```diff
@@ -34,8 +34,8 @@
     from wandb.sdk.artifacts.artifact_manifest_entry import ArtifactManifestEntry
     from wandb.sdk.internal import progress
 
-# This makes the first sleep 1s, and then doubles it up to total times,
-# which makes for ~18 hours.
+# Sleep length: 0, 2, 4, 8, 16, 32, 64, 120, 120, 120, 120, 120, 120, 120, 120, 120
+# seconds, i.e. a total of 20min 6s.
```

Comment:
It also adds jitter, right? So could just be ~20 min.

Reply:
Jitter is 0 by default.